src/coord.jl

```diff
@@ -67,7 +67,7 @@ function first_concrete_aesthetic_value(aess::Vector{Gadfly.Aesthetics}, vars::V
     for var in vars
         for aes in aess
             vals = getfield(aes, var)
-            vals === nothing && continue
+            Gadfly.isnothing(vals) && continue
```

Comment:
qualifying `isnothing` with `Gadfly` is not really needed here, is it? it's done throughout src/coord.jl.

Reply:
Any binding defined locally already shadows one defined in a loaded module (e.g. Base) unless the loaded binding is resolved first. So unless there is a variable inside of this function called `isnothing`, it does not need qualification.

Reply:
This is all within the module `Gadfly.Coord`. `using Compat` is only within the parent `Gadfly` module, so to use `isnothing` here, we either have to add `using Compat` to this module or qualify usage of `isnothing` with `Gadfly.isnothing`. Stylistically, probably better to just add `using Compat` I suppose.

Reply:
Resolved by switching from `isnothing` to `=== nothing`